packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -90,9 +96,12 @@ export class GuxFormFieldColor {
       <GuxFormFieldContainer labelPosition={this.computedLabelPosition}>
         <GuxFormFieldLabel
           position={this.computedLabelPosition}
-          required={this.required}
+          required={
+            this.indicatorMark === 'optional-only' ? false : this.required
+          }
```

Comment:
Based on my comment about which property should take precedence below I think this could just stay the same as before like `this.required=this.required` and then inside `renderOptionalIndicator` the conditional would be `if(this.indicatorMark && !this.required` , the required would take precedence over the indicatorMark if both were present. Let me know what you think.